.azure/pipelines/helix-matrix.yml

```diff
@@ -26,11 +26,13 @@ jobs:
     timeoutInMinutes: 240
     steps:
     # Build the shared framework
-    - script: ./build.cmd -ci -nobl -all -pack -arch x64 /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log
+    - script: ./build.cmd -ci -nobl -all -pack /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log
       displayName: Build shared fx
-    - script: .\restore.cmd -ci /p:BuildInteropProjects=true
+    - script: ./build.cmd -ci -nobl -restore -noBuild -projects src/Grpc/**/*.csproj
       displayName: Restore interop projects
```

Comment:
Nit: is this comment correct still?  Looks like we are just building grpc now with this step?

Reply:
This is now correct. The comment was wrong before because it would restore the default `--all` projects, plus the gRPC (Interop) projects.